runtime/oti/j9port.h

```diff
@@ -377,6 +377,7 @@ typedef enum J9ProcessorArchitecture {
 	PROCESSOR_PPC_P8,
 	PROCESSOR_PPC_P9,
 	PROCESSOR_PPC_P10,
+	PROCESSOR_PPC_P11,
```

Comment:
Shouldn't we start by adding `OMR_PROCESSOR_PPC_P11` in `omrport.h`?

The code in OpenJ9 that uses these literals should probably be removed in favour of processor detection code that already exists in OMR.

Reply:
i did that as well:  https://github.com/eclipse-omr/omr/pull/7512